silk/models.py

```diff
@@ -148,6 +148,8 @@ def garbage_collect(cls, force=False):
         requests = cls.objects.order_by('-start_time')
         if not requests:
             return
+        if len(requests)-1 < target_count:
```

Comment:
Couldn't we combine this line with the line above on 149? I know it lengthens the if check with an or but ensures only one return statement.